README.md

```diff
@@ -38,9 +39,9 @@ pull-requests so you may simply disable this action for run in such cases (via `
 When you want to run action (say with `dry-run` set to `true`) prevent failing you may simply set `check-repo`
 to `false` too.
 
-**NOTE**: You should avoid set both `check-repo` and `dry-run` to `false`.
+**NOTE**: You should avoid setting both `check-repo` and `dry-run` to `false`.
 
-Usually you don't needed set `publish-delay` because this action ever checks availability of published
+Usually you don't need to set `publish-delay` because this action never checks availability of published
```

Comment:
Never?

Reply:
This action checks version availability and tries to download package of corresponding version for each dependency.